.gitmodules

```diff
@@ -39,5 +39,5 @@
 [submodule "bold"]
 	path = bold
 	url = https://github.com/EspressoSystems/bold.git
-	branch = sync-v3.5.6
+	branch = 55c6b6d7daacb856dd877c1ce7c870d7d7b831cf
```

Comment:
can we not use the tag name here?

Reply:
I just figured out we need to have it start by `release-`

Reply:
[157d44b](https://github.com/EspressoSystems/nitro-espresso-integration/pull/607/commits/157d44bf5f2d85e8164ba05f374d8be6d9d6520f)